staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -965,6 +965,22 @@ type PodSecurityPolicySpec struct {
 	// privilege escalation.
 	// +optional
 	AllowPrivilegeEscalation bool `json:"allowPrivilegeEscalation,omitempty" protobuf:"varint,16,opt,name=allowPrivilegeEscalation"`
+	// AllowedHostPaths is a white list of allowed host paths. Empty indicates that all host paths may be used.
```

Comment:
@bgrant0607 👼  whoops, that's definitely noted in the [api changes docs](https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md#edit-typesgo). 

I'll update the docs for my feature and create a ticket to track the others. That sound good?